video/out/cocoa_cb_common.swift

```diff
@@ -115,15 +115,8 @@ class CocoaCB: Common {
     }
 
     override func updateICCProfile() {
-        guard let colorSpace = window?.screen?.colorSpace else {
-            log.sendWarning("Couldn't update ICC Profile, no color space available")
-            return
-        }
-
-        libmpv.setRenderICCProfile(colorSpace)
-        if #available(macOS 10.11, *) {
-            layer?.colorspace = colorSpace.cgColorSpace
-        }
+        super.updateICCProfile()    
+        libmpv.setRenderICCProfile(colorSpace!)
```

Comment:
never use a forced unwrapping in swift if not necessary. always use a guard, if let or ? instead of !. otherwise it will only end up in hard crashes.